content/c-sharp/concepts/Conditionals/conditionals.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'conditionals'
+Description: 'Compares inputs and gives an boolean output depending on it being True or False.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Dev'
+  - 'App Dev'
+  - 'Game Dev'
+Tags:
+  -
+CatalogContent:
+  - 'Learn C#'
+---
+
+## Explanation
```

Comment:
Refer to the [concept template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md) for help with structure